common-util/src/main/java/gov/cdc/etldatapipeline/commonutil/json/CustomJsonGeneratorImpl.java

```diff
@@ -41,9 +41,9 @@ private static ArrayNode generateFieldsArray(Object model) {
 
                 fieldNode.put("type", getType(field.getType().getSimpleName().toLowerCase()));
                 fieldNode.put("optional", (!fieldName.equals("public_health_case_uid")
-                        && !fieldName.equals("observation_uid") && !fieldName.equals("organization_uid")
-                        && !fieldName.equals("patient_uid") && !fieldName.equals("provider_uid")
-                        && !fieldName.equals("notification_uid")));
+                        && !fieldName.equals("act_uid") && !fieldName.equals("observation_uid")
```

Comment:
Suggestion: We can put all the primary keys in a list and use a not contains method.